robozonky-strategy-natural/src/main/java/com/github/robozonky/strategy/natural/LoanTermCondition.java

```diff
@@ -23,7 +23,7 @@
     private static void assertIsInRange(final int months) {
         if ((months < LoanTermCondition.MIN_TERM) || (months > LoanTermCondition.MAX_TERM)) {
             throw new IllegalArgumentException("Loan term must be in range of <" + LoanTermCondition.MIN_TERM + "; "
-                                                       + LoanTermCondition.MAX_TERM + ">.");
+                                                       + LoanTermCondition.MAX_TERM + ">, but was " + months);
```

Comment:
I find this kind of exception "input must be X but was Y" more helpful. I caught an edge case in natural-strategy-setup using this.

Reply:
You're right. I don't do this, because I'm too lazy. :-)